Sources/SmithyXML/Writer/NodeInfo.swift

```diff
@@ -5,7 +5,7 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-public struct NodeInfo {
+public struct NodeInfo: Equatable {
```

Comment:
`NodeInfo` gains fields and methods for handling XML namespaces.

`NodeInfo` is also now `ExpressibleByStringLiteral` which makes much of our generated code simpler and easier to read.